src/client/src/containers/SelectableCard/index.tsx

```diff
@@ -95,20 +93,9 @@ const SelectableCard = ({
             <CardTitle title={title} />
           </div>
         </div>
-        {isPreview &&
-          isFrameworkSelection &&
-          selected &&
-          (option.internalName === "Flask" ? (
-            <DependencyInfo
-              frameworkName={"Flask"}
-              installed={dependencies.python.installed}
-            />
-          ) : (
-            <DependencyInfo
-              frameworkName={"NodeJS"}
-              installed={dependencies.node.installed}
-            />
-          ))}
+        {isPreview && isFrameworkSelection && selected && (
+          <DependencyInfo frameworkName={option.internalName} />
```

Comment:
Just for curiosity, I don't know where do we get the `option` object from?

Reply:
It's grabbed from the store in `SelectBackendFramework` and `SelectFrontendFramework` and then passed down to here, it contains information about the frameworks like author, longdescription, version, licenses, etc